### DISCUSSION TITLE
[Feature] Button/job to refresh meta?

### DISCUSSION THREAD

**Discussion by bcsteeve** [immich-app/immich#2733] [Feature Request]
### The feature

I uploaded my files, then I realized some were missing geotagging or I wanted to rotate the picture, or whatever.  Directly on the server, I use exifinfo (or whatever) to update it, but it doesn't reflect in Immach, even if I refresh the browser.

I'm guessing it has stored the data in the database upon import, yeah?  It would be nice to have a way to refresh.

Workaround:  Delete the images and re-upload them.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
In the admin panel there is a jobs section where you can rerun tasks.

**Reply by bcsteeve** [reply]
Could you clue me in as to which job does what I'm asking?  I did look there.   Extract Metadata sounded like a possibility, but it didn't seem to do anything (although, maybe I didn't refresh the screen after doing that... I'll try)

**Reply by lbouriez** [reply]
Did you ever found out ? I used exiftool to put some Date after noticing my pictures were missing it, despite rerunning the jobs, they stay displayed in the date of discovery...

**Reply by bcsteeve** [reply]
No. I couldn't say if this is still an issue or not, as I did what I wrote as a workaround and deleted and re uploaded my manually altered photos. 

It could be that you changed the "wrong" date.  It seems to me that there are several different dates used by different manufacturers and softwares when it comes to exif 

I can't help further as it has been over a year since I stumbled through it and I never did fully grasp what I was doing lol.  Good luck.

**Comment by Vitani** [comment]
It appears that in the `admin/queues` page you can manually run the `metadata-extraction` job, but this does the entire library (which could be hundreds of thousands of files), it would be nice if you could refresh an individual image, or select a few and do just those files.
